src/app/entity-groups/research-entities/metadata-representations/project/project-item-metadata-list-element.component.html

```diff
@@ -0,0 +1,12 @@
+<ng-template #descTemplate>
+  <span class="text-muted">
+      <span class="item-list-project-title">
+        <span [innerHTML]="mdRepresentation.firstMetadataValue(['dc.title'])"></span>
+      </span>
+  </span>
+</ng-template>
+<ds-truncatable [id]="mdRepresentation.id">
+  <a [routerLink]="[itemPageRoute]"
+     [innerHTML]="mdRepresentation.getValue()"
+     [ngbTooltip]="mdRepresentation.allMetadata(['dc.title']).length > 0 ? descTemplate : null"></a>
```

Comment:
Same comment

Reply:
I've replaced it with `dsoNameService.getName(mdRepresentation)`